src/sp/it/pl/main/Guide.kt

```diff
@@ -82,7 +82,7 @@ class Guide(guideEvents: EventSource<Any>? = null) {
     private val proceedAnim = anim(millis(400)) { popupContent.opacity = -(it*it-1) }
     val hints = Hints()
 
-    init {
+    fun init() {
```

Comment:
This breaks Guide lifecycle, byt requiring manual initialization - noone is going to remember and adhere to that. Such changes are very dangerous and should be done only after careful consideration.